utils/helper.sh

```diff
@@ -54,8 +47,8 @@ function version_gt() {
 function validate_version_confluent_cli_for_cp() {
   validate_version_confluent_cli_v2 || exit 1
 
-  VER_MIN="1.40.0"
-  VER_MAX="1.43.1"
+  VER_MIN="2.3.1"
```

Comment:
@DABH @ethanruhe do we know what versions of CLI CP 7.1.0 will ship with?  (min/max)

Reply:
Not official of course, but this is reported on my current setup:
```
➜ $CONFLUENT_HOME/bin/confluent version
confluent - Confluent CLI

Version:     v2.3.1
Git Ref:     bc90de55
Build Date:  2021-12-10T21:02:09Z
Go Version:  go1.16.3 (darwin/amd64)
Development: false
```

Reply:
It'll ship with 2.6.0 and CP will be officially interoperable with "2.6.0 through the most recent version." 

It is a little odd to not have 2.0.0 compatible with any CP version, but we know the early 2.x releases had some meaningful bugs. Those were cleaned up in the intervening releases, so I think pinning the earliest officially interoperable version to the one that comes packaged is the best thing for customers.

Reply:
Actually, [2.6.1](https://github.com/confluentinc/cli/releases/tag/v2.6.1)

Reply:
Thanks!  I'm leaving at 2.3.1 min for now since that is what is shipped with the nightlies.